packages/open-truss/src/configuration/engine-v1/Frame.tsx

```diff
@@ -107,6 +102,102 @@ export function Frame(props: FrameContext): React.JSX.Element {
   }
 }
 
+function RenderFrames(
+  props: FrameContext,
+  signals: Signals,
+  reRender: () => void,
+): JSX.Element[] | undefined {
+  const renderType = props?.frame?.renderFrames?.type
+  if (renderType === 'inSequence')
+    return renderInSequence(props, signals, reRender)
+  return renderAll(props)
+}
+
+function renderInSequence(
+  props: FrameContext,
+  signals: Signals,
+  reRender: () => void,
+): JSX.Element[] | undefined {
+  const {
+    frame: {
+      frames,
+      renderFrames,
+      view: { component },
+    },
+    globalContext: { FrameWrapper, workflowId },
+    configPath,
+  } = props
+  if (renderFrames?.type !== 'inSequence') {
+    throw new FrameError(`This error should never occur`, component, configPath)
+  }
+
+  const frameLevel = `${configPath}.frames`
+  const frameToRender = getCurrentFrameCursor(workflowId, frameLevel)
+
+  const nextFuncName = parseSignalName(renderFrames?.next)
```

Comment:
This creates a `next` function that moves to the next frame. If we want to make a `back` function we would add something similar here.